### DISCUSSION TITLE
Study Plan to handle FDS and Fire Safety Engineering effectively

### DISCUSSION THREAD

**Discussion by IngSillo** [firemodels/fds#12190] [Q&A]
Dear all,

I've been studying FDS for a few months following the suggestion I already received on the Discussion Group on Google Groups. I'm doing it during weekends and afterhours.

I was asked to prove:

- the structural stability of a warehouse during evacuation;
- understand at what time does the building collapse would collapse
- a safe and effective evacuation of civilians working in the warehouse before the structural collapse.

The request came from an old civil engineer who has no time or experience with FDS but he's enough experienced to say the model is feasible. 

I've just finished reading the FDS User Guide, but so far I'm convinced that FDS is the only tool I must read and learn. I'm now going to read the other fundamental documentation (Technical Guide, Validation Guide, Smokeview manuals). 

I will set up a meeting with the local firefighting brigade to understand which standard they know, follow, and accept.

Would you mind suggesting any other references, software, and tools I should understand to demonstrate the structural stability of a warehouse and the safety evacuation? Up to now, I've bought the super interesting "SFPE handbook of fire protection" which is very high level so any other suggestion is welcome.

The main objective is to be able to produce and defend the model while presenting it to the competent authority by the end of January.

Do you believe the time is enough? 

Kind regards

A

**Comment by emanuelegissi** [comment]
Just my two cents...

I am afraid the time is not enough.

Designing fire safety with quantitative engineering methods is not just
using a software.

I suggest taking a formal course. I do not know where you are based, there
is ample offer of such training both in Europe and in the US.

Emanuele

Il dom 15 ott 2023, 17:18 IngSillo ***@***.***> ha scritto:

**Reply by IngSillo** [reply]
Emanuele,

many thanks for your reply, I'm in Italy by the way, so if you have any suggestions, or if you are going to offer a formal education, I will be happy to attend as much as I can as time and money. I saw you have been involved in FDS development. It would be an invaluable opportunity.

I'm now employed as a Project Manager for a renewable developing company, I have almost 5 years of experience in a multinational structured company dealing with complex permitting procedures (VIA, AIA, VAS etc. you might have heard).
  
I am quite convinced that it is still possible to make a first model for the case I have to handle by the end of January, or maybe I can postpone the deadline and hire a consultant to teach me just in case. 

It has happened to me many times in the environmental field to see the public body reasoning a little obtusely, without comparisons to international standards. I'm sure given your expertise you can understand what I mean.

I already know how to structure the geometry on FDS with Namirial and/or Thunderhead applications. It remains for me to figure out which ways to do the validation and use the appropriate codes for stability.

Would you be so kind as to share some material in Italian? At least to give me a reply on the standard to be considered while designing. I'm scared to read and use something like SFPE that might be unknown by local authorities.

My intention is to become a reliable and inexpensive subject matter expert for simple applications within a year or so to complement my main business. I am in no hurry.

Grazie ancora del tuo tempo!

Alberto

**Reply by IngSillo** [reply]
*edit

With "Subject matter expert" che ho scritto al volo intento un tecnico locale che possa portare l'approccio FSE in uno studio di provincia senza alcuna ambizione accademica ovviamente

**Comment by mcgratta** [comment]
* the structural stability of a warehouse during evacuation;
* understand at what time does the building collapse would collapse
* a safe and effective evacuation of civilians working in the warehouse before the structural collapse.

FDS cannot help you answer any of these questions.

**Reply by IngSillo** [reply]
Many thanks for your reply. I know that FDS won't provide a solution to engineering problems which instead would require technical expertise which I still do not possess.

I have been asked to study and understand FDS and related software by a reputable (local engineering firm) which would then review anything I would draft.

However, I received useful information from Dr. Gissi above, who also indicated a course that I might be able to follow next year. 

If you could provide any additional references to books or online courses (from reputable institutions) I'd be glad to study.

regards.

**Comment by drjfloyd** [comment]
People's lives will depend upon your ability to correctly apply FDS. This is much more than just setting up the geometry:

Do you have the expertise to develop appropriate design fires?
Do you have expertise in predicting egress?
Do you have the expertise to take FDS predictions of thermal conditions and evaluate the performance of the structure?
Do you have sufficient background in fire dynamics that you will be able to know if you have a valid simulation or not? FDS will be perfectly happy to simulate whatever inputs you give even if those inputs are bad.
Do you have a working knowledge of the applicable codes and standards for fire protection and structural design?
For any of the above where you lack formal training and experience will you be working under the direct supervision of someone who is?

I don't know what liability laws in Italy are but if there was a loss of life or property due to a structural failure during fire resulting form inadequate analysis as a result of a lack of qualificaitons on your part, there could be criminal and/or financial liability. 

Think very carefully if you have the qualificaitons to do this work.

**Reply by IngSillo** [reply]
Dr. Floyd,

Many thanks for your reply. 

The more I spend time on the FDS Guide and related Github, the more I feel honoured to discuss with worldwide experts that spend their time developing FDS and teaching people.

To reply to your points. I will never sign or submit anything to the authorities if I do not have a complete understanding and knowledge of what the models are saying and without having a clue of the physics behind.

Happy to see that someone still poses ethical questions in a world dominated by business. 

I've already got a way to live with my job. I'm not here to make easy money.

There are actually two drivers that brought me here:

1. During Uni, I developed my Engineering Master Degree thesis without having proper supervision due to the fact I've been involved in a project that actually never started. Today, a little older and experienced, I'm willing to learn  a subject that would provide me with an invaluable technical expertise.
I must admit that it would also tickle my hypertrophied ego.

2. To be as realistic as possible from a business risk standpoint: I'm now employed in the renewables sector. It is an astonishing moment for the sector, especially in Europe. The market is booming. However, I can't predict the future. In 2010's while I was a young student, the market boomed for a while, and then it crushed with several lost jobs due to weird political decisions. I sought and found this opportunity for a collaboration with the small engineering firm I mentioned in the initial question. This might lead me to start my own small and reputable engineering firm in the future, and at the same time have a backup plan in case renowables market would crash again. 

To reply to your last concern, I will not sign anything since the Engineer who offered me the collaboration he's going to be the one with the power to sign as he's enlisted in the Italian Fire and Safety chartered professional and I'm not.

Again, many thanks for your time and reply. It is a privilege to me discussing with you all.

Regards

Alberto

**Reply by johodges** [reply]
Agree with Jason's comments and concerns above. The first step to gaining the knowledge required to accomplish your objective is to understand the principles of performance-based design. The SFPE Engineering Guide to Performance-Based Fire Safety Design (https://www.sfpe.org/publications/resources/pbdguide) is a good starting point on this topic. SFPE occasionally offers a course which expands on the topics presented in the guide. After understanding the principles of PBD, you will need to work under the oversight of someone with experience in this area.

**Reply by IngSillo** [reply]
Thank you for the reply I'll have a look at the SFPE Guide and online resources.
